### DISCUSSION TITLE
fa/manager/installation-and-setup/How-to-install-different-versions-of-Hiddify-Manager/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#3983] [Content Help کمک به اصلاح محتوا]
# fa/manager/installation-and-setup/How-to-install-different-versions-of-Hiddify-Manager/

The proxy manager platform.

https://hiddify.com/fa/manager/installation-and-setup/How-to-install-different-versions-of-Hiddify-Manager/

**Comment by alborz2103** [comment]
سلام من اپدیت کردم سرور از کار افتاد 
پاک کردم دباره نصب کردم بک اپ نصب نمیشه رو اپدیدت جدید 
نسخه های قدمی  هم نصب نمیشه لطفا کمک کنید

**Comment by Aref6725** [comment]
bash <(curl i.hiddify.com/v10.7.0)
